geonetwork/content.md

```diff
@@ -12,50 +12,130 @@ The project is part of the Open Source Geospatial Foundation ( [OSGeo](http://ww
 
 %%LOGO%%
 
-# How to use this image
+## How to use this image
+
+GeoNetwork 4 uses an Elasticsearch server to store the index of the documents it manages so **it can't be run without configuring the URL of the Elasticsearch server**.
+
```

Comment:
Can we start with a complete example right here on how to run the stable release? People can read further to try different configuration options.